src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final String type = "trapezoid";
+    private final int top;
+    private final int bottom;
+    private final int height;
+
+    public IsoscelesTrapezoid(int top, int bottom, int height, String color) {
+        this.top = top;
+        this.bottom = bottom;
+        this.height = height;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (top + bottom) * 0.5 * height;
```

Comment:
Using magic numbers directly in calculations is not recommended. Consider defining a constant for the number `0.5` with a descriptive name that indicates its purpose.